www/cherokee/Portfile

```diff
@@ -1,27 +1,66 @@
 PortSystem      1.0
 PortGroup       old_openssl 1.0
+PortGroup       github      1.0
 
+github.setup    cherokee webserver 1.2.104 v
 name            cherokee
-version         1.2.101
-revision        2
-set branch      [join [lrange [split ${version} .] 0 1] .]
+revision        0
+
 categories      www
 platforms       darwin
 maintainers     nomaintainer
 license         GPL-2
 
+homepage        https://www.cherokee-project.com/
+
 description     Cherokee web server
+
 long_description \
   Cherokee is a relatively new HTTP server designed to be as\
   performant as Apache and lighttpd, but much easier to \
   configure and maintain.  It ships out-of-the-box SCGI \
   support, making it ideal for deploying dynamic websites.
-homepage        http://www.cherokee-project.com/
-master_sites    ${homepage}download/${branch}/${version}
 
-checksums       md5     ef47003355a2e368e4d9596cd070ef23 \
-                sha1    b27f149c7d7111207ac8c3cd8a4856c05490d136 \
-                rmd160  dd3dedc352ba17bdcefd8e200143b8ffa19ad035
+checksums       rmd160  f5271b4b6f6be3cf70a88a3933fd90597862b289 \
+                sha256  5cbd00ff48503eaf90356b2975e311c02977f9166927e57fc23f541a109efd98 \
+                size    5312997
+
+set ck_wwwroot  ${prefix}/var/www/htdocs
+set ck_cgibin   ${prefix}/var/www/cgi-bin
+
+post-extract {
+    reinplace -W ${worksrcpath} \
+        "s|/Library/WebServer/Documents|${ck_wwwroot}|g" configure.ac
+
+    reinplace -W ${worksrcpath} \
+        "s|/Library/WebServer/Documents|${ck_wwwroot}|g" admin/util.py
```

Comment:
The whole point of the `-W` flag is to specify a parent directory in which you will modify multiple files. Since both of these reinplaces make the same change, use a single reinplace to do it.

Reply:
Resolved